code/datums/gas_mixture.dm

```diff
@@ -11,8 +11,8 @@ What are the archived variables for?
 	(carbon_dioxide*SPECIFIC_HEAT_CDO + (oxygen+nitrogen)*SPECIFIC_HEAT_AIR + toxins*SPECIFIC_HEAT_TOXIN)
 
 #define MINIMUM_HEAT_CAPACITY	0.0003
-#define QUANTIZE(variable)		(round(variable,0.0001))
-
+#define QUANTIZE(variable)		(round(variable,0.0000001))/*I feel the need to document what happens here. Basically this is used to catch most rounding errors, however it's previous value made it so that

```

Comment:
I don't think it will? Once again I'll see in testing. QUANTIZE is rarely used as a TEST though, it's mainly used when it comes time to actually transfer the gases, and so for the most part it just silently disappears them while still running all the procs.